dags/longitudinal.py

```diff
@@ -40,6 +40,17 @@
     uri="https://raw.githubusercontent.com/mozilla/telemetry-airflow/master/jobs/addon_recommender.sh",
     dag=dag)
 
+ensemble_recommender = EMRSparkOperator(
+    task_id="ensemble_recommender",
+    job_name="Extract the addon data for each Telemetry user to DynamoDB",
+    execution_timeout=timedelta(hours=10),
+    instance_count=20,
+    owner="vng@mozilla.com",
+    email=["telemetry-alerts@mozilla.com", "vng@mozilla.com"],
+    env={},
```

Comment:
This is the object that populates the environment variables on the cluster -- looks like your script below expects `date` to be set